test/E2ETests/E2ETests/Fixtures/FunctionAppFixture.cs

```diff
@@ -55,7 +55,12 @@ public async Task InitializeAsync()
                     _funcProcess.StartInfo.ArgumentList.Add("HelloFromQuery");
                     _funcProcess.StartInfo.ArgumentList.Add("HelloFromJsonBody");
                     _funcProcess.StartInfo.ArgumentList.Add("HelloUsingPoco");
+                    _funcProcess.StartInfo.ArgumentList.Add("HelloWithNoResponse");
+                    _funcProcess.StartInfo.ArgumentList.Add("PocoFromBody");
+                    _funcProcess.StartInfo.ArgumentList.Add("PocoBeforeRouteParameters");
+                    _funcProcess.StartInfo.ArgumentList.Add("PocoAfterRouteParameters");
                     _funcProcess.StartInfo.ArgumentList.Add("ExceptionFunction");
+                    _funcProcess.StartInfo.ArgumentList.Add("PocoWithoutBindingSource");
```

Comment:
More of a forward-looking question, but how are we going to test both non-ASP.NET functions _and_ ASP.NET functions with this e2e app as only one mode is really supported at a time?

Reply:
Will need to work through that. While we can make the logic conditional on what we pass into the process, we might want to consider a separate app.